housekeeper/store/tag_filters.py

```diff
@@ -0,0 +1,25 @@
+from enum import Enum
+from typing import Callable, List, Optional
+from sqlalchemy.orm import Query
+from housekeeper.store.models import Tag
+
+
+def filter_tag_by_name(tags: Query, tag_name: str) -> Query:
+    """Return tag by bundle name."""
+    return tags.filter(Tag.name == tag_name)
+
+
+class TagFilters(Enum):
```

Comment:
I think we decided on keeping the enum in singular :)

Reply:
Lets change the names of all filtering classes in another PR :)